Explorer/Assets/DCL/Multiplayer/Connections/RoomHubs/MessagePipedMutableRoomHub.cs

```diff
@@ -0,0 +1,45 @@
+using DCL.Multiplayer.Connections.Messaging.Hubs;
+using DCL.Multiplayer.Connections.Messaging.Pipe;
+using LiveKit.Internal.FFIClients.Pools;
+using LiveKit.Internal.FFIClients.Pools.Memory;
+using LiveKit.Rooms;
+
+namespace DCL.Multiplayer.Connections.RoomHubs
+{
+    public class MessagePipedMutableRoomHub : IMutableRoomHub
+    {
+        private readonly IMutableRoomHub origin;
+        private readonly IMutableMessagePipesHub messagePipesHub;
+        private readonly IMultiPool multiPool;
+        private readonly IMemoryPool memoryPool;
+
+        public MessagePipedMutableRoomHub(IMutableRoomHub origin, IMutableMessagePipesHub messagePipesHub, IMultiPool multiPool, IMemoryPool memoryPool)
+        {
+            this.origin = origin;
+            this.messagePipesHub = messagePipesHub;
+            this.multiPool = multiPool;
+            this.memoryPool = memoryPool;
+        }
+
+        public IRoom IslandRoom() =>
+            origin.IslandRoom();
+
+        public IRoom SceneRoom() =>
+            origin.SceneRoom();
```

Comment:
Could be just get-only properties. Also, should it then implement `IRoomHub` interface as well?

Reply:
ah right, missed it

Reply:
Does not matter really, up to you

Reply:
In this case I prefer methods, because they are accomplished with command methods from IMutableRoomHub interface